docker/test/integration/resources/syslog-client/tcp/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM ubuntu:latest
```

Comment:
Please use a specific version instead of latest as the latest tag could refer to another version later.

Reply:
Good idea. I've changed this in https://github.com/apache/nifi-minifi-cpp/pull/1294/commits/26053fde5f0a6d4ed055ee98558d9653fd89254a